lib/manager/reducers/projects.js

```diff
@@ -224,7 +224,8 @@ const projects = (state: ProjectsState = defaultState, action: Action): Projects
       }
       return update(state, {
         all: {[projectIndex]: {feedSources: {[sourceIndex]: {$merge: {
-          feedVersions: versions.sort(versionSort)
+          feedVersions: versions.map(() => null), // versions.sort(versionSort), // TODO remove
```

Comment:
We can probably remove this now?

Reply:
Yes, thanks, removed in https://github.com/ibi-group/datatools-ui/pull/776/commits/0ce316a2a47261d6690897583c5e05626859e9da.